circuits/cpp/barretenberg/cpp/src/barretenberg/honk/pcs/ipa/ipa.test.cpp

```diff
@@ -84,8 +84,10 @@ TEST_F(IPATest, Open)
 TEST_F(IPATest, GeminiShplonkIPAWithShift)
 {
     using IPA = IPA<Params>;
-    using Shplonk = shplonk::SingleBatchOpeningScheme<Params>;
-    using Gemini = gemini::MultilinearReductionScheme<Params>;
+    using ShplonkProver = shplonk::ShplonkProver_<Params>;
```

Comment:
having to declare all of these is not ideal and we should probably do something about it (if possible), maybe similar to the macros we defined for stdlib tests: https://github.com/AztecProtocol/aztec-packages/blob/master/circuits/cpp/barretenberg/cpp/src/barretenberg/stdlib/primitives/safe_uint/safe_uint.test.cpp#L12C1-L12C1